share/doc/xml/srcflux.xml

```diff
@@ -69,11 +68,30 @@
 	psfmethod argument. Extended sources should use psfmethod=ideal,
 	which makes no correction, or psfmethod=psffile and provide
     an image of the surfrace brightness as the psffile parameter.
-        The other options are discussed below in the psfmethod parameter 
+        The other options are discussed below in the psfmethod parameter
      and the "PSF Correction" section.
       </PARA>
+      <PARA title='A fully Baysian computation'>
+	This tool is based on aprates, which performs a fully Baysian computation,
+	thus the result is not a (frequentist) confidence interval,
+	but should correctly be called credible interval.
```

Comment:
I would not get into the discussion of the difference between the correct/incorrect names, so confidence or credible interval in the documentation. Suggest change lines 76-77 to:
"thus the result is a credible interval (confidence interval for the frequentist calculation), which is CONF parameter."